frontend/src/pages/afdelingen.tsx

```diff
@@ -1,52 +1,65 @@
-import dynamic from 'next/dynamic';
-import {useMemo} from 'react';
-import {fetchAfdelingen, fetchFallback} from '../utils/backend';
-import Afdeling from '../models/Afdeling';
-import Subheader from '../components/Subheader';
-import AfdelingenTable from '../components/AfdelingenTable';
-import Banner from '../components/Banner';
-import Main from '../components/Main';
-import {GetStaticPropsResult} from 'next';
-import {revalidate} from '../utils/revalidate';
-import HeadPage from '../components/HeadPage';
+import dynamic from "next/dynamic";
+import { useMemo } from "react";
+import { fetchAfdelingen, fetchFallback } from "../utils/backend";
+import Afdeling from "../models/Afdeling";
+import Subheader from "../components/Subheader";
+import AfdelingenTable from "../components/AfdelingenTable";
+import Banner from "../components/Banner";
+import Main from "../components/Main";
+import { GetStaticPropsResult } from "next";
+import { revalidate } from "../utils/revalidate";
+import HeadPage from "../components/HeadPage";
 
 interface Props {
     pageBanner: string;
     afdelingen: Afdeling[];
 }
 
 export default function AfdelingenPage(props: Props) {
-    const AfdelingenMap = useMemo(() => dynamic(() => import('../components/AfdelingenMap'), {ssr: false}), []);
-    
-    return <div>
-        <HeadPage title="Afdelingen" description="Overzicht van alle afdelingen van ROOD" url="https://roodjongeren.nl/afdelingen"/>
-        
-        <Banner title="Afdelingen" background={props.pageBanner} compact/>
-        <Main>
-            <div className="container">
-                <Subheader>Overzicht</Subheader>
-                <p className="my-4">
-                    Wij hebben {props.afdelingen.length} afdelingen verspreid over Nederland.
-                    Kijk op de kaart of er een afdeling bij jou in de buurt is, of begin er één!
-                </p>
-                <AfdelingenTable afdelingen={props.afdelingen}/>
-            </div>
-            <div className="container">
-                <Subheader>Kaart</Subheader>
-            </div>
-            <AfdelingenMap afdelingen={props.afdelingen}/>
-        </Main>
-    </div>;
+    const AfdelingenMap = useMemo(
+        () =>
+            dynamic(() => import("../components/AfdelingenMap"), {
+                ssr: false,
+            }),
+        []
+    );
```

Comment:
Dit neemt wel erg veel ruimte in beslag. Zou de lambda ook niet op een new line hoeven?

Reply:
we kunnen de line length verhogen van prettier volgens mij. iets van 90 doen?

Reply:
hmm nee het heeft niet met line length te maken, maar ik vind t ook niet super erg eigenlijk